browser/extensions/api/settings_private/brave_prefs_util.cc

```diff
@@ -9,6 +9,7 @@
 #include "brave/components/brave_wallet/browser/buildflags/buildflags.h"
 #include "brave/components/brave_wayback_machine/buildflags.h"
 #include "brave/components/brave_rewards/common/pref_names.h"
+#include "brave/components/ipfs/browser/buildflags/buildflags.h"
```

Comment:
I think you're missing a dep for the buildflag and possibly the component as well

Reply:
I think I already have a dep here:
<img width="857" alt="Screen Shot 2020-09-29 at 3 23 39 PM" src="https://user-images.githubusercontent.com/831718/94605935-c7171d80-0267-11eb-8820-78f281b314f8.png">